### ISSUE TITLE
apollo-server-fastify is using deprecated fastify packages

### ISSUE THREAD

**Issue by tetchel** [apollographql/apollo-server#6381]
Starting my server, I get the following messages:
```
(node:3652446) [FST_MODULE_DEP_FASTIFY-ACCEPTS] FastifyWarning.fastify-accepts: fastify-accepts has been deprecated. Use @fastify/accepts@3.0.0 instead.

(node:3652446) [FST_MODULE_DEP_FASTIFY-CORS] FastifyWarning.fastify-cors: fastify-cors has been deprecated. Use @fastify/cors@7.0.0 instead.
```

These two dependencies in `apollo-server-fastify` need to be updated to point to the new package locations.

`fastify-cors` -> https://www.npmjs.com/package/@fastify/cors
`fastify-accepts` -> https://www.npmjs.com/package/@fastify/accepts

https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-fastify/package.json

**Comment by glasser** [comment]
Thanks — @SimenB has filed a PR to fix this at #6366. I just have one question to resolve before we can accept that change.

**Comment by glasser** [comment]
Fixed in #6366.
